### DISCUSSION TITLE
Switch names of Venus and Mercury

### DISCUSSION THREAD

**Discussion by Michael-NL** [Stellarium/stellarium#2950] [Q&A] [answered]
Hi, I'm new to github, so I hope I'm posting my issue here correctly:

I'd like to switch the names of Venus and Mercury in Stellarium in such a way that those names are switched in the translation I'm using, which is Dutch (NL). Is there any way of doing so? Either by changing them overall or in the translation only.
So the planet that is commonly known as Venus will be labeled "Mercurius" (= Mercury in Dutch) and the planet that is commonly known as Mercury is labeled "Venus".

Kind regards, Michaël.

**Comment by alex-w** [comment]
Why you want to get it?

**Comment by gzotti** [comment, marked as answer]
While this sounds absurd to want, see section 9.3.3 in the User Guide for a private solution.

**Reply by Michael-NL** [reply]
Not so absurd, as possibly (and I'd say: in all probability) these names have been interchanged in earlier times.
Apart from other references, in the Apocalypse of St. John, the _Morning Star_ is mentioned (in the letter to Thyatira, Ap. 3;28) which clearly refers to Mercury, esoteric name for the second half of the evolution of Humanity and Earth.
Thx for the hint, it works!

**Reply by gzotti** [reply]
While I know that Mercury in some context can be named Morning Star, this does not require such confusing name swap. You may want to call one of them Lucifer in biblical context, then. But you are free to name them as you wish on your screen.
